### PR TITLE
Kotlin android example

### PR DESCRIPTION
**Pull request by swerky**

### CODE REVIEW

File: android-kotlin-client/.idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
```
Comment (bchapuis) [inline android-kotlin-client/.idea/.gitignore:2]: is it a common practice to commit .idea directories in kotlin projects?

File: android-kotlin-client/app/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    id ("com.android.application")
+    id ("org.jetbrains.kotlin.android")
+    id ("com.google.protobuf")
+}
+
+android {
+    compileSdk = 32
+
+    defaultConfig {
+        applicationId = "ch.heigvd.android_kotlin_client"
```
Comment (bchapuis) [inline android-kotlin-client/app/build.gradle.kts:17]: Should we use the same naming convention for all of our jvm packages? (it hasn't been formally set yet)
`ch.heigvd.examples.grpc.kotlin`

File: android-kotlin-client/.idea/.gitignore (hunk 1 above)
Comment (bchapuis) [inline android-kotlin-client/.idea/.gitignore:2]: is it a common practice to commit .idea directories in kotlin projects?

File: android-kotlin-client/app/proguard-rules.pro [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
+# You can control the set of applied configuration files using the
+# proguardFiles setting in build.gradle.kts.
```
Comment (bchapuis) [inline android-kotlin-client/app/proguard-rules.pro:3]: Does this mean that we could remove this file (as we can manage proguard rules with gradle)?

File: android-kotlin-client/app/src/androidTest/java/ch/heigvd/android_kotlin_client/ExampleInstrumentedTest.kt [hunk 4]
```diff
@@ -0,0 +1,24 @@
+package ch.heigvd.android_kotlin_client
```
Comment (bchapuis) [inline android-kotlin-client/app/src/androidTest/java/ch/heigvd/android_kotlin_client/ExampleInstrumentedTest.kt:1]: package name?

**Review by bchapuis — APPROVED** [review]

File: android-kotlin-client/.idea/.gitignore (hunk 1 above)
Comment (swerky) [reply android-kotlin-client/.idea/.gitignore:2]: In the official example, they do push a part of the .idea.

**Pull request merged**